sel4-support.h

```diff
@@ -0,0 +1,172 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+/*
+ * Copyright 2023, Technology Innovation Institute
```

Comment:
I had a second thought on this -- isn't this copied from seL4 source tree so copyrights should be assigned to that Data61?

Reply:
Replaced with `data61` License.